source/release-notes.txt

```diff
@@ -10,6 +10,23 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.43.4
+----------------
+
+*Released July 1, 2024*
+
+Bug Fixes:
+
+- Prevents application from hanging when selecting ranges too quickly on the 
+  "Schema" tab (:issue:`COMPASS-8048`).
```

Comment:
[Optional suggestion]

Maybe link "Schema tab" to this page: https://www.mongodb.com/docs/compass/current/schema/